app/build.gradle.kts

```diff
@@ -57,15 +61,51 @@ android {
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_11.toString()
     }
+    sourceSets {
+        getByName("main") {
+            res {
+                srcDirs(
+                    "src/main/res",
+                    "src/main/java/org/grapheneos/apps/client/ui/container/res",
+                    "src/main/java/org/grapheneos/apps/client/ui/mainScreen/res"
+                )
+            }
+        }
+    }
 }
 
 dependencies {
+    /*Default android*/
     implementation("org.jetbrains.kotlin:kotlin-stdlib:1.6.10")
     implementation("androidx.core:core-ktx:1.7.0")
     implementation("androidx.appcompat:appcompat:1.4.0")
```

Comment:
just FYI, appcompat has updated to 1.4.1 few days ago: https://developer.android.com/jetpack/androidx/releases/appcompat#1.4.1

Reply:
updated.